Automattic/sensei
# Add changelog links

### Changes proposed in this Pull Request

* It adds the changelog links to extensions which contains update, and to the update notification banner.
  * Extensions not installed, or installed without update will show the link to the extensions page.

### Testing instructions


* Enable the feature flag: `define( 'SENSEI_FEATURE_FLAG_EXTENSIONS_MANAGEMENT_ENHANCEMENT', true );`
* Make sure you have some extensions installed, others not, and others with pending updates (you can update the `Version` in the [plugins header](https://developer.wordpress.org/plugins/plugin-basics/header-requirements/) to simulate they need an update).
* Make sure the extensions with updates will have a link to the changelog, while the others will have a link to the extension page.

cc @donnapep to confirm if the described behavior makes sense.
